content/en/platform/corda/5.0-dev-preview-2/introduction/corda4-differences.md

```diff
@@ -10,16 +10,14 @@ section_menu: corda-5-dev-preview2
 ---
 
 ## Relaxed PKI
-Corda 4 required that all Corda networks use the custom PKI (x509 extensions) dictated by Corda.
-Membership of a Corda Network was granted through Corda operating as a certificate authority with the signing of a CSR tying a legal identity to a public key granting membership of a network.
-This was achieved through the use of the Identity Manager and Network Map services run as part of the CENM product.
+Corda 4 required that all Corda networks used the custom PKI (x509 extensions). Membership of a Corda Network was granted by Corda operating as a certificate authority. The signing of a CSR tied a legal identity to a public key, thereby granting membership of a network. This was achieved by the use of the Identity Manager and Network Map services, which are run as part of the CENM product.
```

Comment:
Hi Colin I also did some edits here. I put "X.509" instead and also spelled out "CENM".